src/stylesheets/elements/_form.scss

```diff
@@ -60,9 +60,7 @@ input {
   &[type="radio"]:not(:checked),
   &[type="checkbox"]:checked,
   &[type="checkbox"]:not(:checked) {
-    width: 2px;
-    height: 2px;
-    opacity: 0;
+    display: none;
```

Comment:
I believe the element needs to be visible in order to meet accessibility standards, confirm @vinternet-pds

Reply:
I've just amended this with margin 0.